.env

```diff
@@ -1,5 +1,6 @@
 TF_VAR_name="ct-platform"
 TF_VAR_domain="ctdapp.net"
+TF_VAR_google_region=europe-west6
```

Comment:
Sure, but I considered interesting to add it there, so it can boost the development while not impacting the CI pipeline.
As in the CI Pipeline is being overwritten there is no problem to have it here as well.